css/src/components/popover.scss

```diff
@@ -0,0 +1,40 @@
+$popover-background-color: $body-background !default;
+$popover-border: $border-width solid $border-white-high-contrast !default;
+$popover-border-radius: $border-radius !default;
+$popover-shadow: $box-shadow-heavy !default;
+$popover-width: 224px !default;
+
+.popover {
+	display: inline-block;
+	position: relative;
+
```

Comment:
There's a csstricky way to dismiss the popover when the user clicks off of it:

```scss
       // line 10
	&[open] .popover-summary:after {
		content: '';
		position: fixed;
		top: 0;
		right: 0;
		left: 0;
		bottom: 0;
	}
```

Reply:
Spoke offline, we agreed to scrap this for now, since it's more of a modal pattern.

Reply:
As agreed over the teams - not going to do that.

Reply:
Normally I would agree- this could be done as a separate step. In this case we need this selector for the behavior to more closely match the existing Docs .dropdown component.